cmd/removeEnumerator.go

```diff
@@ -78,9 +81,11 @@ func newRemoveEnumerator(cca *cookedCopyCmdArgs) (enumerator *copyEnumerator, er
 	finalize := func() error {
 		jobInitiated, err := transferScheduler.dispatchFinalPart()
 		if err != nil {
-			if err == NothingScheduledError {
+			if err == NothingScheduledError && !cca.dryrunMode {
 				// No log file needed. Logging begins as a part of awaiting job completion.
 				return NothingToRemoveError
+			} else if cca.dryrunMode {
```

Comment:
Minor: it's more readable to simply rearrange the order of the if statements.

Ex: 
if cca.dryrunMode {...}
else if err == NothingScheduledError {...}